presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/docker/DockerFiles.java

```diff
@@ -130,4 +141,9 @@ public static Path createTemporaryDirectoryForDocker()
         }));
         return temporaryDirectoryForDocker;
     }
+
+    public interface HostDirectory
```

Comment:
The implementation is tied to a host directory, but the abstraction defined by the interface could be better described as "file provider" or "resource provider", not imposing implementation constraints.

in such a case this could be
```
interface ResourceProvider {
 String getPathOnHost(String resourceName)
}
```

Reply:
```
interface ResourceProvider {
 String getPath(String resourceName)
}
```
?

Reply:
Can be, although getPathOnHost was intentional.
You could also make the method reutrn `Path`, but that would not play as nice with TC APIs.

up to you